src/store.js

```diff
@@ -1,29 +1,21 @@
-/* eslint-disable no-underscore-dangle */
+import { configureStore } from "@reduxjs/toolkit";
 
-import { createStore, applyMiddleware, compose } from "redux";
+import accountReducer from "./slices/AccountSlice";
+import bondingReducer from "./slices/BondSlice";
+import appReducer from "./slices/MainSlice";
+import pendingTransactionsReducer from "./slices/PendingTxnsSlice";
+// reducers are named automatically based on the name field in the slice
+// exported in slice files by default as nameOfSlice.reducer
```

Comment:
Why does it make sense to have the default export of something like `MainSlice` be `appReducer`. seems non-intuitive.

Reply:
I wasn't sure what to do here.  In all the projects and documentation I looked at, I never saw RTK with AppSlice.  We were using state.app in many places and I wasn't sure I wanted to change that since we're probably used to it by now so I made MainSlice define state.app. If you think it should be different Im down to change just let me know